langserver/lsp.go

```diff
@@ -0,0 +1,95 @@
+package langserver
+
+type InitializeParams struct {
+	ProcessID             int                `json:"processId,omitempty"`
+	RootPath              string             `json:"rootPath,omitempty"`
+	InitializationOptions InitializeOptions  `json:"initializationOptions,omitempty"`
+	Capabilities          ClientCapabilities `json:"capabilities",omitempty`
```

Comment:
**[govet]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
struct field tag `json:"capabilities",omitempty` not compatible with reflect.StructTag.Get: bad syntax for struct tag pair